patches/chrome-installer-mac-BUILD.gn.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/installer/mac/BUILD.gn b/chrome/installer/mac/BUILD.gn
+index 1425094d2cb2605fbc9ce11d68276471322d5640..23029d1e900dee50cc55fc7de4ac6235a50e3098 100644
+--- a/chrome/installer/mac/BUILD.gn
++++ b/chrome/installer/mac/BUILD.gn
+@@ -15,7 +15,7 @@ group("mac") {
+   ]
+ }
+
+-_packaging_dir = "$root_out_dir/$chrome_product_full_name Packaging"
```

Comment:
what is the purpose of this change?

Reply:
In order to change the path of the packaging directory to have underscores rather than spaces in the directory name.

Reply:
I understand what it's doing, but why do we need to do that?